api/data_source.go

```diff
@@ -96,10 +96,10 @@ type DataSource struct {
 	SslCa    string
 	SslCert  string
 	SslKey   string
-	// HostOverride and PortOverride are only used for read-only data sources for user's read-replica instances.
-	HostOverride string            `jsonapi:"attr,hostOverride"`
-	PortOverride string            `jsonapi:"attr,portOverride"`
-	Options      DataSourceOptions `jsonapi:"attr,options"`
+	Host     string            `jsonapi:"attr,hostOverride"`
```

Comment:
is this intentional to keep as hostOverride to avoid breaking frontend? Same below

Reply:
Yes. And we need to change frontend and switch dual-write to single write to data source.